Module4HM4_EF/Module4HM4_EF/appsettings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+
+  //"ConnectionStrings": {
+  //  "PetContext": "Data Source=localhost;Initial Catalog = master;user id=sa;password=Doja270503;Integrated Security=False;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
```

Comment:
Yes, As I said - NEVER commit sensitive data into the repository. Now I can find it in the changes =)